### ISSUE TITLE
DPS for Build Pricer calculating wrong

### ISSUE THREAD

**Issue by lucasagostini** [PathOfBuildingCommunity/PathOfBuilding#5323]
### Check version

- [X] I'm running the latest version of Path of Building and I've verified this by checking the [changelog](https://github.com/PathOfBuildingCommunity/PathOfBuilding/blob/master/CHANGELOG.md)

### Check for duplicates

- [X] I've checked for duplicate issues by using the search function of the [issue tracker](https://github.com/PathOfBuildingCommunity/PathOfBuilding/issues)

### Check for support

- [X] I've checked that the calculation is supposed to be supported. If it isn't please open a feature request instead (Red text is a feature request).

### What is the value from the calculation ingame?

DPS for build pricer is not calculating DPS for main hand weapon for RF. If I try to search and let sort by default, all the 10 pages of items that are displayed have lower DPS than my current weapon.

### What is the value from the calculation in Path of Building?

In POB, when I try to use price builder to search for weapon 1 with RF as main skill, all weapons that appear lower my skill DPS.

### How to reproduce the issue

Select RF as the main skill. 
Open Build Price
Search for Weapon 1
Verify that all weapons actually lower skill dps.

### Character build code

```shell
eNrtPdly20a2z8OvQKkq98Va0CsAX3umtFuxJSukl8QvrgbQEGGBAIXFkjw1_35PNwgSoACKWybJ3EkqCgn0Od1nX7oBvvrHwygyvss0C5P49Q7aN3cMGXuJH8Y3r3c-fjjbs3f-8ffeq2uRD98HR0UYqTt_7_3tlf5sRPK7jF7vOGTHyEV6I_NPFSryFVCNRZwPZRJfim9Jep74r3eukljuGK6I_TCvvnmRyLIrMZKvdy5FKgpfpjuGyDwZ-8ezWz8XNzcyjUWR7xgjEcaDxLuV-XmaFOPXO3zH-B7K-8vEh5EXl9fv-x9qs4dxfXZY_d9eXUfiUaaDXORGBn9e7xwCE8SNfBMC-u8iKgCPuXPQPnYwltJ_fth1Kk-DQHp5-F0ep2F-PBSxJ5-HW3XsZRHl4TgKFdsm49E-64J48wQ5MjvRf0hyEZ1cD55fSDkymXGPOoRjvk8tC5mIdC7nc5gPAaw-xwqQRxFIYqn1qdEXN3GYy6WHXydhlsQrUb8GGcdFFIFNLTXNcTJyw7hJ8bLTJEnkJ_fxbJJ9QroGH6ZSvA9K5e0LPyyyKRjuBBrkM_0jJu4a1pd39ZGIdS75RD7MhnXPCwgbIxHqGnkRz9QTm3QBwvpIRFD3Cr8nObi7JRXk9M31DKtFCLb3LYtTm3Or04cMH7PQE9GleAhHxQiM94O4lbMJkU0cpwv4XXgzzGNQri5oizisU1BnYSq7AG2CHKtb1SJ_rRnBLyVZFyRhtr2A0mDmz6hlogWu23upRl_E3oyJHC1C_DFOZSbT7zWfTzE3F4H0pZeogOJGcmmY2TTXMoXgN9NAx9znyDGRhYlJLGvxxDcynsz-OCOQMra_EO6dlN7wHMJqX-RLhJ1LEYuaTPlCdqvBdXYvxNrCbLIkwDzb6L5pcdPmDuMMI2sRkg624X1nEdSKTDuNZXrzOBiGMvJr7speZny1tmMxXgZU870OX-c_xsvN2MaSpYhbkS-fRbpENtWH7GzFNZVRWcFMx3KwJcQXAlzJvN-AsSx7n3ZGCxg-NbwZCGf7bBHIVOmWUjYAeCKaJUTyXWT14IQodxYrTAmwnK1KSL0AwJdPssnufDj5prLhaEW4w3SUFLOMgSJmsoV0lAANle_OR6oIeyJGWsP8wmuEdKdzXUcRlCDLJuqwsChqg6Cd2niY58K7PUn8G7nSLKtDDIrxGLyoEv6ycCo76MsszGb-1qbPD34Pllv3YWRhFjE_AX5-8PwE3c5umh0tPcscxJOp0MLEZm4azpYYPT9Ft61XheYl-MYRREFdo0I9XCsJcbd8oAZZqgDRA5tlTre5J_ew-KFqC2SrjYYUcKYgnUtJZfzjcWn8jeFLTXAa-0WqjGHpOeYh6tNUVv4hHIHnzLITkQsjkyL1hu9A0K93dmrfzkQUuWD7cFVBvTrQ3Rb16WI0TtLciESWH3peUsT5G5ENX-8w5lk4gGKAeR7jPncIFgR78JV6xLWDwMGubVJnR4P2pYhGwPPj8qtarvBymZa4bOF6LkaOS5h0TQu5Dpdm4AeYu44tHciofNdiQantr45F5GWatIt4XORGrLs1ozDzvrpFEKiWDJCWp7qjdHp2dnr84eLT6YQbdZDsNoyir3ExclUjo_w_qO1k5EBqt2xkhZuVH1_vfArl_UBBnchchBFIyEuiSIwz6b_eCUSUwcwhfNRjBsB2L1-ATY-CsqMMBF24ZgO6MZ0-yDQHeiGx8dJQdq5rev-ZRZUTqqRHaU8XNtV86UZUBpNjJXydlHVwSne2urGodlMnOermAlhQXRF1zjy5-wwn8sexzCDEh0HoKcNbLPIPMLoctYAvngc26z0ukPckJHbj0F2oLgTlzW7gssvUBT25u4CruqfVydXybjf4ifREJ-3lzW7gaZ6fxLpv2o5lOmoBpqsk1koORnMYRip-dUr2NJLTId0I3-dDmZZBsRPTJfioashCw0lDt8i7zbg2YgGvdIeig0PqXjdoWWx30KDuLfBEjZKzg6H1Md2oyiqt05EtAu0v8Kf9xa60yv07OF_eXUB_VQJ1kD65vWDpOg3rFP0kp1vkGiDj73YN6mY3cGlaT1HkabEkBm0GJzKQ4MAW2sF0zAIHnBfxCfjKfAFDl0RVBrTD70nol9XGRtg0ke3IZpxaCVfpX1r5tjJGXY1tyvq5Im0r7J-0Vztcf23IM4jAZ79ZkBUsh2lav7yB1FRtvSTRZgiftJE3ojPJMxH7upW0NKGvDqrU-JXORjIjg9T9XI6yo0cooM6Uv53b0hGaBZN0NVfJbxOmUj9fBqKI1PVfChGF-aMqSmpX35XbsnGSjqZbB4BqmNwrJSoxquQoAxf67l155zDKJ8jUdBVlJUsmC9JkIn1tclHvxB7OVq0p1usOYy8qfHkRT2pLWE0YqXLDVStTu8yqOT41qCxKVCdQijGwHu3M4y2h9cR_ewXrm4CfR4krIlzhLqnG5k7jNprOoTBdwJRnkRiNhBsqYneMu5LqC53waA7uGDdypL5fQl3hQ6F2cJEDTw4UYw70euBTKxIthznSdJU2WaKqdFSLam4NB8tRhp4j7MKTJ1DCrU9TmmS5G4Xx7ToUzYAPtiWpiboeF2km35dJ5qqkTRW-BcnyxL2RD3lSeMMJaa9Klj1vCRrNJraAf2dbuIISWqd-Zc9ibdVpRbI8f0twYwK_bf05g7IXwrougTfQoFY0KxiIBjdK-GzrRKoqIle9wrXpm8ewPGkV5L_NOoAX31VmtoFtoGcZe-g-ZpmIjtP1o0QLirpIn2HrRRyoI0-RoeC3pi8nEtRwFMaTTH5NytqxLK8yTfi_kuIswWGdCr6fNOwOoWhan83dqJbn9aWApSeRnLaxFhCpibgMY5DKXEq3vEO6DXMQrpuoqmAen0a1bpApRiONZoZ-9UijcRhda6yvXu0q_fs0842EqjPfzKU5z6a-YSo_pOtEiSpFnEewgjtTsIYG3nbsU0gPYx9EKKv2_dpRsBvXCqQqJAaUjIZCY9R3DLaZFuvG9Iek6gGvnxq3I9Kn_ZajV5WxRVZz5c_rKv0D8xr8p01sjhL_0ai6nk9cAdpaiOqrvWyZFJkyyrW9QTuWFfg7RWBoDNu2kNMgCD29u7O28jxBsQJ1U9iDbfnvybouYi-VIpP-UZGqAwkbO71nEK5A8wTBij5vacpPI328QURniVdkm8i1HdEq0q0wGCWKbZPaF-PQn2zQrU1mC5IVSBzch0FuHAZBFHqrOPc_tGhF6M_r3HW6-bvWHpdJlMt4MJRRtLZbb8OxQrWhoY0S_D-Ktx_kaCyzvNpPXZO77ViW5-8E3qhv2f6RHN5eUvIpzMVGTfknCJbn6hR0WTeOlnRxR1GS-JfiJvQ28HItSFZw44dpmtzMDrHUlWXycSBzvWVV7lSpjx9SKasNKY0ETTaD4Ev96bwLvScEWqGam4-T8xSvd_6J1DHdXY4R_9fuPwnGFO1Si1OsvlEHW7twxTbhGyMMWbuEYn2PU2ryXeY4RMFhG9u7CDOk7yD4wpHpWPCNWsgBHDYntvpGsWPvEpOZ-F9AeOKrrS1k2g7ZJY4NU2PHpM4uUzDMomSX2hagsk26i4kJi0GWxWG1lMF1YsIlWDksc9dCXH20nV3MLcx2KQwEOpDFdjFzLBMuIAf-OogRAMfIhIHURkAPsoBitXaghijcavmwAlgoJxaG4YRSQOZgRuEzkLpLsENgOmJSoJrZHP5yE2jgMJfiJVAJ6JHDAT0FDICdwGeAVdQghRlRNa2NEHwGdsBCuEkVs4lNFBc42lUXYBbTJGppjuIwQZzuMsV1WCHDu8zGgIDaCjGiJuPqLzVhCLdAOgym3C0lwIiN4AqxFKtNYpuADBDtEoY5jMdEzUqAOZWYGVKf1fpAQSggxiZAEgsuMJNQxUvg1C4F7aC7jgN0Y46B0RN9suAeIRQYCnoCaJF6_g1WamOsyDYdGOhgqpADScBgB9aCKEgRWYTCRxOWWKoL8IHsWpTCVe6ArAnnFhCNbROECLRRRSKsTWsrc4BMii0gCsFfkINa1i6yLaS0QnEUMxs4ijRfSzUmtgOaQyyCuZrTVBJWhsCxCbxAsCLAZjm2Ih0UFKbiIDGHK6KRw-xdrf-Y2xz45CBgFIbrsAzThCQyBwutPW-rvIE3s8jS0D_235XubJjn4-zlwcH9_f3-WOTDJJAPYST3vWR0MAYgMPM97bn2FNqDQ_jn6Ob0KDi-uxrxswfZ71-djewj9vUs-xxc2wPyRdrUdPeGYV-cf_zl4-dPj8l79_AwSAdfyPmZ5Kffzh6_8pvLTO69_06QfTRyfrw9HF9_ubgO7vdIMDixHTT6dPGFXF6Hx0d79nv7aM9yiodP3y6d4BpwWCff7oK95LdL_EMU_V9uKL6z4x9ucpV_-xJc3dwWZ2Tw_upi6N_dmw4d2Wdvzh7v_NOrYf7mOviWnh-9e3-SZ4Pb-18G2ejYe_dlTDO7n9CbcSF_fBj_9j0VljXgH9_nYfGVpKOj7AsTb823Zyzzz34--vTNFPHl2a-fT0ejC_fB88b3Nw_00_DboF-Eh-cf4p8fv1rReHh16_HsLiYP9u1pbt5-OX3IwzdvB78Gn7zf3p2cf3scjO2LO_7Ll8PDuLgL7g5dh9Nff_v12PzF_G1w_OPBfxT5ccTM8Pzr3cV18bkYjh37lw_fju_vX5feupLiq_Kh52wSocpv2t0pmWsPBbE9LwNK9cRT20BwIbWBfBYWZvjhC3h6HRRUKFAfrpJc6nvqYvVFBwp1EndyfnmQpyqD-JEko9909FWHC95IkV-K8fS8hLo7OaRAJgcTcpGfhBA9Un16pYpnauCv1ZHrVzo0ToKS-lwdkigyOZBeEvvlpqm-XDu_oIaWZxd4aRZ9kUIAfWn0D_unPUiBbqCMM3SI7R2n4ShLYuNneS-j3sc4vCukcXHy0pDClsxyfJ9CdLAZ-CficFMGyEcWB59vmabgLvhg2-O2YC5yLfBKzA4s7Dqu7_f0KjTRLw0b9_Snvrx7aWDeuxiNocIK8-ylgXrHSZoW41z65ZIMT8RxkhuuhLRNF2JwB1b4mBS9F-wnI08mtbahTggbapvAmD0S3kP8J5XuldW97rJMhs_dUad31XMwBggwvJVG2RM07sN8aEwffDDK_KBn1yFnz_UY-vzwjACtVJeJ3xcxLC9Vf9WDx6w8MIMrtescQZ4dQZ8dUT1mofOqOXWwWtQBCua8ACadZpnSxAUKYVHkCgSRk2EWoEC6NgkCKqW0pGM6tsU9ykzTkoHJAuwFLHA8HthYWhAIfYIaCmFZdR0wey9sJVcBWfzsdOkCUVr1G6PywI-hytveC0S0iuiHSozyvKLQdC0nJPQsg7coxlYhtZlsAqqWi7T3fqwU1pNjCFp12TDKRWAHjDE_gKQisCRjKiJjSLGlzVziEeQhJJkjuETgOBFzXdvHnDOGPY81ZUN6k5NRYKtmb-IjXxpHe_BvzY652bRjatalMusWTWTWsCF9wq-6c-j7GcykxEaZoXahKguHK3pk1iP288YLwncUSKUQ6rAwXFOX9KqNJNAqpjWpXEFZOario4dxY4F5ERvV8U_lfk5jOQpBKf_ppSIANfrXC2xqTavpZbtH-vPo2_NuY8EI_uwI69kRdrfe46bef7y6-OXjaW8gIERG0f9AmMv-NzPOUqjxekeA1gcZvYWIPukK9MotBFBYiC7l5yOQ4-QRaXDV4GT2EVQDvfL49wTspUF5r34g_CmQCVly3dSwS00noD7zEYIMmlrEox73HBd8nKTCJxxSXqjMEAOLk5x6tgsO01SVAeJ-EDRNjbab2vlef69fMzXmNE3tBf2p8pczO5s5O1Bm0M1JRINx-th00-aw1WmOLzB6Bntt-srW6rePywBezjo5_vMfZgVtGkyaGnx5eH5x3DsNgkIVG8ZJ-F1tjaogZZxB2XKrvJF6kADSjLp-2Yh5gUWcgDNw1MSDMOu7kIhZ1AXXDgUw8S2OAsfl0vNcRFwoz1zQbGH5jEqvmXeBi57qF3I6fbfZoyDRVD0LDMpwOFJNFWPy3DkETKhw68pSPZAOvMllj4GmASXTi2KsvJ5vXMRKHfLosdeXI3UGxdAHEjPlSyF__ZNHYdomy0uRJd4Q1LxyR8DbNP8xE2c-lMahK6IkbkRnB3kuhSobJGZhz0bStBzTc23LxL5ley4RgnuYmD7CPsEBg0qbIR87geMTCNJO02Wwhsuo-Yi5cDwNVB-V0O6HMp4cN8xA0sIbGkERRT31zIWBp3f0MMi0DQFB7U0IWThItAx-jz2K6mpQelnDL9SjjkbZCPsdk1fWmhcVEaTjYTbqDXQxplfyKJ8mr24AKarlWyhwXe55ni-I6TpeIHwHEdt0TVv1jTzfcoktOWRHgY0oQhiZLnxwm17bRgssqcxkIS-R8Q2UEupMxEWcg3sNb3R-rZMdpJMdSHnmkp3KWb5ATGcyJY_BI5N6XqMTXf1SBKlM0EB0n5aOZQzpR6Yrwz9lvrue5NuK2OMhJFlp6OnjNb1B_ngTgpp-CrOG4eHAkdjCIEeBXch2XR8qFMcKmOAmFCjchaBsI8FsRhxHSBM7NrcwJMg2WKmL5qRei8-HTVN7I0CkxuTso1GO6r3gzkKhYbKPnkjtBakllbXqZZZ1TuobPUAp13yhk_3l5d1Wpb5N1XMmxpV67Lk3GEPO7xvl0cZp5sccszPzY7aJGvWS7WHsSiYcyQJpBlC-0sAlmEgEPgG8cUCswPUcDwnKoYQFzXEcRH3hcGYR05mvZVuTuM97n9V_NV9hNXwF7l2U_Y1SmKcPEFRUJQ4BEjzvro6kj8qj7SHUqhSTbsizBcgLZNV9Uu-FY9ecC2NPvXoPNcotraVT3VUFkQ79L2ipi7N2SavGcuuJHfy3JFqmJLJbDOE8DUfGtYx9yK16H4r0rkjA6UHOVkTgc2oq7gkC5T1kjRaGXBEzCGI-KHfgc0QIg8xDEoahLpGBJW0MZYz0EaU-tQLPtETwpF3TFfHwTNKHEWT8oBiZTicgKgtVS-iGnXzIpaLiaTxs7fzMaV-lXo1-gW4X4Cd362G05m6xs8-eetunvYFaQ2Cmwcz8rwavpcHOUmn0OWBUpXwjjx7cFvFtI4uWgliUMcmZCBzkQnBn2JOCQVoniO1A3mzDRd_3Pc4c5lmUIGZLXzoOIdIV88H8D8uiofJWb0SZdpdgwlERK-MAJStfgLBKVv17N43NFi90pJrEOZQ5xnEqs7zXTx4h8Tkq0hu4ls9Csml3hmQbQWFT91ckwJ4L5DLHVpuD1OQgVx-5UAcLRzDCfE4tJnBgM-mZUEP5JkJ2IBkRmJtW7zTywbLVupuSZl0tlvO9o0aThbNmJl8OVYmG3oVRPcPyfAJcAoGWbkPn8LGiS7kZv5LYFHbiioyh-C6NF2Uc7-heNufzJQxQ2eAoUcnBfDf1BWEK1ZwrxU_8FCbN6C5FhaGWUja84GzpyemcI3Se4ILxy-jofx1iZU2ozSOC47sBMUJIr1xivwANa_jDIyhw6_bCIb4LzzMdDpbj-hDuOZSxJpeq1-94PlwGs3CR7xJfBDY4ygCbmLiO9BxmCjm_P9fuDjld2x2S5dwhZ7POU6lKimKo7yAMl7t-c86QoxnAWSrlD9no0z8F-CN7ggi35XBRkoyMTzKM1GYsmHQqwG9m4BHk1HU6vLuPjSGb603e5QG5mW1XX1qGEm43KmJhmQR7xJSYUN-0bIq552CHBxaUP9z3fIkxktRyLASOWG3jebZg1MSeb1PkNbuLNdc6t0_Uh__Om96VNL2rqjCmHccq84MC4q6Acka_7gbqlGbZQnAjl4QLjSYJ1bcnrDD6Qr2SqmfTlp6VSkIn43qIPdn2UbfL5nXV1fyLdlO24edatZq0aPXnRBpQkMS9w5HMh49ZbvTnWtuutJgfeNSRmGHHwiRgJiRynk09jztEcgGpnbQdHzEpJQXrVZWL8JktMMV4zmU1ShJrbn9ksjf3ZBdYlw66mcaM6k2YLYVD2aQrhy0qMF5Q3l6ZTypj3d6rX8e0fV2zYEzx__tao1Xllu3IhyCXMFJtkGY5AelUowPEfArVg--DltmS28wNXMuzTcsJ1Mv-uO9SCJtQJyMoiCH7pHaARGBhm0tqw6DfPXwuWU0w_JPhTTf69FuBwM6UtuoQCrM0g6Hq9rSPP0uTH_JPVXm0tfyrw-Bv48K7jdSxjpv7JL194mssHwlCbHUUxYFwZpuQE1EZqAIRUz-AeGaZgWM5JECYBohZgWCu2shVhyXEvHxpp68hvT1kXKcyCB8MIDIMQtA8EUXJvdT9hUERtN6qUEzvZEa1bVoWBDcKHjdaZ42t6_lNgfKww8QfPfE7lHV36iCuPueUEPvDm85_suR-wQhngUrbracc3skb4T0qV3VWQLLx-TEd6WzQ0I8gTHNCAn6mKyckHNmznFCN7MwJ9dBGTkhcl1u2C3U1tl3kuAHyHCosZHpUEBRwz3c8KBwgbJvc8phJCKXMksJ1bc7c-bDcWm73jSNj7kwDaiaE2LKfSdVI44BR9ebfySG8K6iPwDFOjumU5bqXpN4Q7jUBZ1VGOaBHnKlPnPrat6r-EFMUpbfVZykgxTFOla_OijRNilhv4Je3gyQ16KS5mO3qEl7Zmv1TFYXeli5Zj65Clm5KiqzRxq_1KP_CuznqFVrdR1Wbb9eKwPOpA7rX7sf-O3WgtnxJbvWGrMG9GM9tsBm1s79mtcLn8MzjwEvgWHSUPE-FLw_Kk8AHb46hbs-Hurk3AHXKi9HBKDU_yXMZ8PmV1M4jP7d47QW2QX0bF_EKeI5klG8CX2ZkZAbhLLly3LZysgX5r8FF_HvgoFughW9hHevgaJMNWkWr2tR7FeGWbxLZaAktis03XsEqplG9p6cCsNabkmzJK6CNiadb0Ea0BaugK6rBbDhfTmYbTdim-tvg3OpeuebF6FrL3jSQkJX4Nn1ryCY60y7AzXWfbYxhdedTyyeWlHiNzmkOWE5_4cPw-cenSlB9ss6oPVG1jSSp75-P3932C2s9gbHtaA7e1HD5FtwV2YLxkw0tkW4pCdjUI9QUlG0gV7KyL6wxANHlQGq0IrQOsdvK59dRoBkI3kTefFMDQtsx5G3p71qBWCFaxw-sCbepHFTftKG8bD0f3GrrrcFEP7TbDCbT53irqdaX_TZ88JpqWGMiXo-JZFXB1bsIZBOVoduxPLZx8rWyD6lxwN7E2FdZennacgbQOPBRvVpEP7ytfzEqiYPw5snvP0nVBByEP-S1-kHPfPajUZfSD4tRyy9Gzf_A9wxE_9D3QfsUF9lRkmXPjp3_7fJnAcofNW8ddh0JTw4TfSCrTmz5O-LVT11Z9d_hbhuvzklVL2argMgSMNVv-lUwbDFI8xcRZ2DO5AfuXh1UEnx1MP_D8P8Hd1BWWA==
```

### Screenshots

![Capturar3](https://user-images.githubusercontent.com/11837075/207361905-91b0a169-5e07-4975-9462-8c407c11b988.PNG)

**Comment by QuickStick123** [comment]
You can fix this temporally by increasing the minimum weight to 100,000 on the trade search.

**Comment by lucasagostini** [comment]
@QuickStick123 where can I change the weight? I tried to look for it in the window and don't see that information. Is it in a specific file?

**Comment by LocalIdentity** [comment]
@lucasagostini Change the number that is in the same position as this red box on the trade site
![image](https://user-images.githubusercontent.com/31035929/207373621-bb0573b5-26b2-460f-a1a8-ad29ab2c74b6.png)

**Comment by lucasagostini** [comment]
I believe that either the way the weighting is calculated is wrong (for my build) or that something is not right. With 100000 it finds no items, with 50000 it finds 8 weapons and they are all worse than mine (that is not very good).

**Comment by lucasagostini** [comment]
here is the link to help: 
https://www.pathofexile.com/trade/search/HC%20Ruthless%20Sanctum/5Br2vDnTa

**Comment by chx** [comment]
I _think_ but I am not sure, the problem here effects all DoT builds, it should use -- but it doesn't -- Combined DPS or such. Everything it finds makes my pConc build worse.

**Comment by QuickStick123** [comment]
Fixed by https://github.com/PathOfBuildingCommunity/PathOfBuilding/pull/5340
